cardano-cli/src/Cardano/CLI/Commands/Ping.hs

```diff
@@ -22,3 +23,16 @@ data PingCmd = PingCmd
   , pingOptsGetTip :: !Bool
   }
   deriving (Eq, Show)
+
+wellConfigured :: PingCmd -> Maybe String
+wellConfigured
```

Comment:
verbs in function names are more understandable I think
```suggestion
getConfigurationError :: PingCmd -> Maybe String
getConfigurationError
```

Reply:
Fixed.